src/host/getset.cpp

```diff
@@ -1317,6 +1317,35 @@ void ApiRoutines::GetConsoleDisplayModeImpl(ULONG& flags) noexcept
     return STATUS_SUCCESS;
 }
 
+// Routine Description:
+// - A private API call for changing the screen mode between normal and reverse.
+//    When in reverse screen mode, the background and foreground colors are switched.
+// Parameters:
+// - fReverseMode - set to true to enable reverse screen mode, false for normal mode.
+// Return value:
+// - STATUS_SUCCESS if handled successfully. Otherwise, an appropriate error code.
+[[nodiscard]] NTSTATUS DoSrvPrivateSetScreenMode(_In_ bool fReverseMode)
+{
+    try
+    {
+        Globals& g = ServiceLocator::LocateGlobals();
+        CONSOLE_INFORMATION& gci = g.getConsoleInformation();
+
+        gci.SetScreenReversed(fReverseMode);
+
+        if (g.pRender)

```

Comment:
When we add support for pass-throughing this sequence, we should make sure to _not_ redraw all here when we're in conpty/VtIo mode

Reply:
Yeah. I realise this is not great, but it was the easiest way to get it working in the Terminal for now. I'd prefer to avoid using pass-through until we have a better way of sharing the conhost code with the Terminal. Reimplementing everything twice makes me want to 😭